<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1090</width>
    <height>655</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Circuit Creator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>900</width>
      <height>600</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>111</width>
      <height>350</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <pointsize>8</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Color</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxColor">
         <property name="currentText">
          <string>black</string>
         </property>
         <item>
          <property name="text">
           <string>black</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>blue</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>red</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>yellow</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>green</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Angle</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBoxAng">
         <property name="maximum">
          <number>360</number>
         </property>
         <property name="singleStep">
          <number>90</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>X coordinates</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBoxCorX">
         <property name="minimum">
          <number>100</number>
         </property>
         <property name="maximum">
          <number>4500</number>
         </property>
         <property name="singleStep">
          <number>50</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Y coordinates</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBoxCorY">
         <property name="minimum">
          <number>100</number>
         </property>
         <property name="maximum">
          <number>4500</number>
         </property>
         <property name="singleStep">
          <number>50</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="btnAddSource">
       <property name="text">
        <string>Add Source</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnAddResistor">
       <property name="text">
        <string>Add Resistor</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnAddLine">
       <property name="text">
        <string>Add Line</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnModify">
       <property name="text">
        <string>Modify</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="frameModify">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>151</width>
      <height>171</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QToolButton" name="btnDown">
     <property name="geometry">
      <rect>
       <x>61</x>
       <y>100</y>
       <width>27</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="arrowType">
      <enum>Qt::DownArrow</enum>
     </property>
    </widget>
    <widget class="QToolButton" name="btnRotate">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>40</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Rotate</string>
     </property>
     <property name="popupMode">
      <enum>QToolButton::DelayedPopup</enum>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonIconOnly</enum>
     </property>
    </widget>
    <widget class="QToolButton" name="btnLeft">
     <property name="geometry">
      <rect>
       <x>22</x>
       <y>70</y>
       <width>27</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="arrowType">
      <enum>Qt::LeftArrow</enum>
     </property>
    </widget>
    <widget class="QToolButton" name="btnRight">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>70</y>
       <width>27</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="arrowType">
      <enum>Qt::RightArrow</enum>
     </property>
    </widget>
    <widget class="QToolButton" name="btnUp">
     <property name="geometry">
      <rect>
       <x>62</x>
       <y>40</y>
       <width>27</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="arrowType">
      <enum>Qt::UpArrow</enum>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxID">
     <property name="geometry">
      <rect>
       <x>51</x>
       <y>10</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::UpDownArrows</enum>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
    </widget>
    <widget class="QPushButton" name="btnOK">
     <property name="geometry">
      <rect>
       <x>52</x>
       <y>140</y>
       <width>41</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
    <widget class="QToolButton" name="btnDelete">
     <property name="geometry">
      <rect>
       <x>95</x>
       <y>40</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1090</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menufile">
    <property name="title">
     <string>file</string>
    </property>
    <addaction name="actionsave_elements"/>
    <addaction name="actionread_elements"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menufile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionsave_elements">
   <property name="text">
    <string>save elements</string>
   </property>
  </action>
  <action name="actionread_elements">
   <property name="text">
    <string>read elements</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnOK</sender>
   <signal>clicked()</signal>
   <receiver>frameModify</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>82</x>
     <y>589</y>
    </hint>
    <hint type="destinationlabel">
     <x>85</x>
     <y>521</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnModify</sender>
   <signal>clicked()</signal>
   <receiver>frameModify</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>85</x>
     <y>380</y>
    </hint>
    <hint type="destinationlabel">
     <x>85</x>
     <y>521</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
